Godeps/Godeps.json

```diff
@@ -7,11 +7,26 @@
 	],
 	"Deps": [
 		{
+			"ImportPath": "github.com/Microsoft/go-winio",
+			"Comment": "v0.1.0",
+			"Rev": "8f9387ea7efabb228a981b9c381142be7667967f"
```

Comment:
Use same version with containerd https://github.com/containerd/containerd/blob/master/vendor.conf#L36 to avoid problems in the future.